packages/api/src/validation/ajvAPIValidator.ts

```diff
@@ -45,13 +45,22 @@ export class AjvAPIValidator<T extends AjvSchema> implements APIValidator {
   }
 
   private validate(
-    { method, path, query, body }: APIValidatorRequest,
+    { method, path, contentType, query, body, params }: APIValidatorRequest,
     response: unknown,
   ) {
+    // helper to make sure that objects are not empty
+    const isDefined = (obj: Record<string, unknown> | undefined) => {
```

Comment:
Nit: the name here implies a boolean return value. This is really mapping empty/falsy objects to `undefined`. Might I suggest a name like `undefinedIfEmpty` instead?

Extremely nitpicky nit, which you should feel totally free to ignore: I think it's clearer to use `null` in situations like this than `undefined`, because the two have subtly different meanings. `null` represents the intentional and definitive absence of a value, whereas `undefined` values are automatically assigned by the environment when an expression extends outside its domain of definition, like "a variable which has been declared but not initialized," "a valid object projection on an uninitialized key", "the return value of a void function", etc. Here we're defining a function with an intentional semantic of "emptiness" (i.e. objects with no keys map to emptiness for our purposes here).

Reply:
Good point, makes sense. I'll go ahead and use null instead!